src/geo/leaflet/leaflet-map-view.js

```diff
@@ -45,8 +45,15 @@ var LeafletMapView = MapView.extend({
       // remove the "powered by leaflet"
       this.map_leaflet.attributionControl.setPrefix('');
 
+      // Disable scrollwheel
       if (this.map.get("scrollwheel") == false) this.map_leaflet.scrollWheelZoom.disable();
+      // Disable keyboard
       if (this.map.get("keyboard") == false) this.map_leaflet.keyboard.disable();
+      // Disable dragging (also doubleClickZoom)
```

Comment:
Do these really add any value? I find the code to be simple enough to be self-explanatory